couchbase_cluster_admin/cluster.py

```diff
@@ -1,7 +1,27 @@
 import logging
+import json
 import time
 
 from .client import BaseClient
+from .exceptions import (
+    AddToNotProvisionedNodeException,
+    BackupPlanCreationException,
+    BackupRepositoryCreationException,
+    BucketCreationException,
+    ClusterJoinException,
+    ConnectToControllerOnJoinException,
+    DeleteAlternateAddressException,
+    IllegalArgumentError,
+    NodeRenameException,
+    RebalanceException,
+    SetAlternateAddressException,
+    SetAuthenticationException,
+    SetClusterNameException,
+    SetGsiSettingsExceptin,
```

Comment:
```suggestion
    SetGsiSettingsException,
```

Reply:
Fixed in 8acf35c34a979ab1f582e04ba3ce7d0c91a62352